### ISSUE TITLE
Dashboard improvement: click through per circuit breaker with more detail

### ISSUE THREAD

**Issue by JorritSalverda** [Netflix/Hystrix#1024]
Some very useful data is left out of the dashboard to make sure you don't suffer from information overload, but it would be nice to be able to click on a circuit breaker and then see the following
- a larger version of the graph
- the settings used by the circuit breaker, particularly the timeout value
- the total percentiles since the applications emitting the streams started

**Comment by mattrjacobs** [comment]
Yes, this seems like a good idea.  It's pretty unlikely this makes it to the top of my priority list, however.  See #1033 for a discussion on the dashboard roadmap.
